packages/python-packages/protocol-stub-generator/protocol/protocol_models.py

```diff
@@ -0,0 +1,796 @@
+            self.add_text(None, "]", None)
+            self.overview_tokens.append(Token("]", TokenKind.Text))
+
+        self.add_space()
+        self.overview_tokens.append(Token(" ", TokenKind.Text))
+        token = Token(self.operation, TokenKind.Keyword)
+        token.set_definition_id(self.namespace + self.operation + "overview")
+        token.set_navigation_id(self.namespace + self.operation + "overview")
+        self.overview_tokens.append(token)
+        self.add_keyword(
+            self.namespace + self.operation,
+            self.operation,
+            self.namespace + self.operation,
+        )
+        self.add_space
+
+        self.add_new_line()
+        self.add_description()
+        self.add_whitespace(3)
+        self.overview_tokens.append(Token("(", TokenKind.Text))
+        self.add_punctuation("(")
+
+    def add_description(self):
+        self.add_token(Token(kind=TokenKind.StartDocGroup))
+        self.add_whitespace(3)
+        self.add_typename(None, self.description, None)
+        self.add_new_line()
+        self.add_token(Token(kind=TokenKind.EndDocGroup))
+
+    def to_token(self):
+        # Remove None Param
+        self.parameters = [key for key in self.parameters if key.type]
+
+        # Create Overview:
+
+        # Each operation will indent itself by 4
+        self.add_whitespace(1)
+        self.overview_tokens.append(Token("  " * 4, TokenKind.Whitespace))
+
+        # Set up operation parameters
+        if len(self.parameters) == 0:
+            self.add_first_line()
+            self.add_new_line()
+            self.add_whitespace(3)
+            self.overview_tokens.append(Token(")", TokenKind.Text))
+            self.add_punctuation(")")
+            self.add_new_line(1)
+
+        for param_num in range(0, len(self.parameters)):
+            if self.parameters[param_num]:
+                self.parameters[param_num].to_token()
+            if param_num == 0:
+                self.add_first_line()
+            self.add_new_line()
+
+            # Add in parameter tokens
+            if self.parameters[param_num]:
+                self.add_whitespace(4)
+                for t in self.parameters[param_num].get_tokens():
+                    self.add_token(t)
+                    self.overview_tokens.append(t)
+
+            # Add in comma before the next parameter
+            if param_num + 1 in range(0, len(self.parameters)):
+                self.parameters[param_num + 1]
+                self.add_punctuation(",")
+                self.overview_tokens.append(Token(", ", TokenKind.Text))
+
+            # Create a new line for the next operation
+            else:
+                self.add_new_line()
+                self.add_whitespace(3)
+                self.overview_tokens.append(Token(")", TokenKind.Text))
+                self.add_punctuation(")")
+                self.add_new_line(1)
+
+                self.add_token(Token(kind=TokenKind.StartDocGroup))
+
+    def to_json(self):
```

Comment:
also wondering if this is used, seems like the to_json on the LLCClientView class takes care of it?